test/support/component_helpers.rb

```diff
@@ -7,54 +7,38 @@ def assert_has_component_title(title, context = nil)
     end
   end
 
-  def assert_has_button_component(text, attrs = {})
-    all(shared_component_selector('button')).each do |button|
-      data = JSON.parse(button.text).symbolize_keys
-      if attrs[:href] && data[:href]
-        next unless text == data.delete(:text) && attrs[:href] == data[:href]
-      else
-        next unless text == data.delete(:text)
-      end
-      data.delete(:data_attributes) if data[:data_attributes].blank?
-      return assert_equal attrs, data
-    end
+  def assert_has_button_component(text)
```

Comment:
Can we make this `assert_has_button`. The tests shouldn't have to care about how the button is rendered (with a component or not).

Reply:
Yeah, that's cleaner.